### PR TITLE
{bio}[foss/2021a] COBRApy v0.26.0 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 4 (1 easyconfigs in total)
node3139.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/21c78bfcb1f51a5349a981f0b30f701d for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3139.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/e441cefb848f9aa4a14addcf5ddfbc69 for a full test report.

File: easybuild/easyconfigs/c/COBRApy/COBRApy-0.26.0-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+software_license = 'LicenseGPLv2'
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python',            '3.9.5'),
+    ('ruamel.yaml',       '0.17.21'),
+    ('SciPy-bundle',      '2021.05'),
+    ('sympy',             '1.8'),
+    ('SWIG',              '4.0.2'),
```
Comment (boegel) [inline easybuild/easyconfigs/c/COBRApy/COBRApy-0.26.0-foss-2021a.eb:25]: @smoretti `SWIG` should only be a build dependency I think?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/COBRApy/COBRApy-0.26.0-foss-2021a.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/c/COBRApy/COBRApy-0.26.0-foss-2021a.eb]: Should be

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @SIB-software!

**Pull request merged**